.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: NIDAQmx package
```

Comment:
Rename this file to PR.yml. We will probably want a separate workflow for publishing, like in https://github.com/ni/measurementlink-python/tree/main/.github/workflows